README.md

```diff
@@ -44,9 +44,19 @@ There are multiple ways to authenticate to use Google Cloud services.
 
 1. When using `gcloud-java` libraries from within Compute/App Engine, no additional authentication steps are necessary.
 2. When using `gcloud-java` libraries elsewhere, there are two options:
-  * [Generate a JSON service account key](https://cloud.google.com/storage/docs/authentication?hl=en#service_accounts).  Supply a path to the downloaded JSON credentials file when building the options supplied to datastore/storage constructor.
+  * [Generate a JSON service account key](https://cloud.google.com/storage/docs/authentication?hl=en#service_accounts).  After downloading that key, you must do one of the following:
+    * Define the environment variable GOOGLE_APPLICATION_CREDENTIALS to be the location of the key.  For example, `export GOOGLE_APPLICATION_CREDENTIALS=/path/to/my/key.json`
+    * Supply the downloaded JSON credentials file when building the options supplied to datastore/storage constructor.  For example, `StorageOptions.builder().authCredentials(AuthCredentials.createForJson(new FileInputStream("/path/to/my/key.json")).build().service()` returns a `Storage` object that has the necessary permissions.
```

Comment:
Example is too long. Maybe should break the lines like this:

* Define the environment variable GOOGLE_APPLICATION_CREDENTIALS to be the location of the key.  For example:
    ```bash
    export GOOGLE_APPLICATION_CREDENTIALS=/path/to/my/key.json
    ```

* Supply the downloaded JSON credentials file when building the options supplied to datastore/storage constructor.  For example:
    ```java
    StorageOptions.builder().authCredentials(
        AuthCredentials.createForJson(new FileInputStream("/path/to/my/key.json")).build().service()
    ``` 
returns a `Storage` object that has the necessary permissions.